system-container/run-system-container.sh

```diff
@@ -0,0 +1,4 @@
+#!/bin/sh
+
+export ANSIBLE_LOG_PATH=/var/log/ansible.log
+exec ansible-playbook -i /etc/ansible/hosts ${OPTS} ${PLAYBOOK_FILE}
```

Comment:
the entire image is mounted as read only when running as a system container.  To have a writeable `/etc/passwd` we would need to bind mount a temporary file.